exercises/if/README.md

```diff
@@ -0,0 +1,12 @@
+# If
+Until now programs ran from the beginning to end following a single logic path.\
+But sometimes we need different flows in our programs. Control flow plays a big
```

Comment:
I'd reinforce that it's not sometimes, it's almost always:
```suggestion
But most of the times, we need different flows in our programs. Control flow plays a big
```

Reply:
Fixed in 81bde052c49ae152c13272cd1e2548bb05859b69